bin/genome-env

```diff
@@ -541,6 +585,8 @@ default_value TESTDBSERVER_DB_USER "genome"
 default_value TESTDBSERVER_DB_PASS "mypassword"
 default_value TESTDBSERVER_URL "https://apipe-test-db.gsc.wustl.edu"
 default_value GE_ENV_NAME "genome-env"
+default_value TESTDBSERVER_DB_HOST "apipe-test-db.gsc.wustl.edu"
+default_value TESTDBSERVER_DB_PORT "5432"
```

Comment:
`TESTDBSERVER_DB_HOST` and `TESTDBSERVER_DB_PORT` defaults shouldn't be in here I don't think.  They get set by `$TESTDB database create --bash` and is information that should either come from the developer, for `-i`, or from `test-db`.